testgrid/cmd/configurator/main.go

```diff
@@ -45,33 +45,34 @@ func (m *multiString) Set(v string) error {
 }
 
 type options struct {
-	creds     string
-	inputs    multiString
-	oneshot   bool
-	output    string
-	printText bool
+	creds              string
+	inputs             multiString
+	oneshot            bool
+	output             string
+	printText          bool
+	validateConfigFile bool
 }
 
-func gatherOptions() options {
+func gatherOptions() (options, error) {
 	o := options{}
 	flag.StringVar(&o.creds, "gcp-service-account", "", "/path/to/gcp/creds (use local creds if empty")
 	flag.BoolVar(&o.oneshot, "oneshot", false, "Write proto once and exit instead of monitoring --yaml files for changes")
```

Comment:
Notice how this clearly states it will cause the program to exit after running once, rather than its normal behavior of running forever.

Reply:
Thanks for the clarification, I misunderstood your comment. Update the flag description and the error message, please let me know if now it addresses your concern.